### DISCUSSION TITLE
Anyway to export note in a format for kindle?

### DISCUSSION THREAD

**Discussion by mo-han** [TriliumNext/Trilium#2786] [Q&A]
in case i'd like to read some notes offline on my kindle device, any suggestions?

**Comment by Duveychan1** [comment]
May I know What kind of note?
And how do you define Offline…
Are you saying off the telecommunication network but wifi??

從 Windows 的郵件傳送

寄件者: 墨焓
傳送時間: 2022年4月13日 9:07
收件者: zadam/trilium
副本: Subscribed
主旨: [zadam/trilium] Anyway to export note in a format for kindle? (Discussion #2786)

in case i'd like to read some notes offline on my kindle device, any suggestions?

**Comment by H0ver** [comment]
Print notes into PDF files, its easy to implement.
